content/en/docs/tasks/deployment/voms/_index.md

```diff
@@ -111,23 +111,32 @@ voms:
     trust-anchors-dir: /etc/grid-security/certificates
 
     # How often should trust anchors (and CRLs) be refreshed
+    # The default value is 86400
     trust-anchors-refresh-interval-secs: 14400
   aa:
     # The VOMS attribute authority host
     host: voms.example
 
-    # The VOMS attribute authority port. Note that this is the port on 
+    # The VOMS attribute authority port. Note that this is the port on
     # the reverse proxy, not the local service port
     port: 443
 
     # The VOMS VO name
     vo-name: ${VOMS_AA_VO}
 
     # Use FQAN legacy encoding, i.e.,
-    # /voms/Role=NULL/Capability=NULL 
+    # /voms/Role=NULL/Capability=NULL
     # instead of
     # /voms
+    # The default value is false
     use-legacy-fqan-encoding: true
+
+    # The VOMS proxy lifetime in seconds
+    # The default value is 12 hours
+    max-ac-lifetime-in-seconds: 43200
+
+    # The optional group label
+    optional-group-label: wlcg.optional-group
```

Comment:
I would write `voms.role` (ref [here](https://github.com/indigo-iam/iam/blob/master/iam-voms-aa/src/main/resources/application.yml#L44))

Reply:
good point. I just reported the default value from the code, but `voms.role` is definitely more sensical.